vagrant/global_vars.yml

```diff
@@ -15,3 +15,9 @@ install_pkgs:
 - kubeadm
 - kubelet
 - kubectl
+# Enables a custom registry
```

Comment:
somehow, i got slightly confused reading this the first time.
could you add a colon at the end of the line to indicate it describes the following line>